src/GithubPlugin/Helpers/LocalSettings.cs

```diff
@@ -0,0 +1,80 @@
+﻿// Copyright (c) Microsoft Corporation and Contributors
+// Licensed under the MIT license.
+
+using System.Runtime.CompilerServices;
+using DevHome.Common.Services;
+using DevHome.Logging;
+using Microsoft.Windows.ApplicationModel.Resources;
+using Newtonsoft.Json.Linq;
+using Windows.Storage;
+
+namespace GitHubPlugin.Helpers;
+public static class LocalSettings
+{
+    private static readonly string _applicationDataFolder = Path.Combine(Environment.GetFolderPath(Environment.SpecialFolder.LocalApplicationData), "DevHome/ApplicationData");
+    private static readonly string _localSettingsFile = "LocalSettings.json";
+
+#pragma warning disable IDE0044 // Add readonly modifier
+    private static IDictionary<string, object>? _settings;

```

Comment:
Why disable the warning?  I thought readonly can be applied to static.

Reply:
Removed.  This was leftover during development before I was actually changing _settings.